.eslintrc.js

```diff
@@ -14,112 +14,37 @@ module.exports = {
         jasmine: true,
         node: true
     },
-    extends: [
-        'eslint:recommended',
-        'plugin:vue/essential'
-    ],
-    // required to lint *.vue files
     plugins: [
-        'vue'
+        '@typescript-eslint',
+    ],
+    extends: [
+        // 'plugin:@typescript-eslint/recommended',
+        'eslint-config-airbnb-base'
```

Comment:
think it would be more ideal to create an Ultimaker.com ruleset. it could be based on airbnb if necessary

Reply:
I surely agree on that, but i also hope we can take out some of the disabled rules in the future..